validation/util/test.go

```diff
@@ -138,3 +138,41 @@ func RuntimeInsideValidate(g *generate.Generator, f PreFunc) (err error) {
 	os.Stdout.Write(stdout)
 	return nil
 }
+
+// RuntimeOutsideValidate validate runtime outside a container.
+func RuntimeOutsideValidate(g *generate.Generator, f PreFunc) error {
+	bundleDir, err := PrepareBundle()
+	if err != nil {
+		return err
+	}
+
+	r, err := NewRuntime(RuntimeCommand, bundleDir)
+	if err != nil {
+		os.RemoveAll(bundleDir)
+		return err
+	}
+	defer r.Clean(true, true)
+	err = r.SetConfig(g)
+	if err != nil {
+		return err
+	}
+	err = fileutils.CopyFile("runtimetest", filepath.Join(r.BundleDir, "runtimetest"))
+	if err != nil {
+		return err
+	}
+
+	r.SetID(uuid.NewV4().String())
+	stderr, err := r.Create()
+	if err != nil {
+		os.Stderr.WriteString("failed to start the container\n")
+		os.Stderr.Write(stderr)
+		return err
+	}
+
+	if f != nil {
+		if err := f("test"); err != nil {
```

Comment:
I don't understand this hard-coded `test`.  Don't we want to pass the bundle path, like [we do for `RuntimeInsideValidate`][1]?

[1]: https://github.com/opencontainers/runtime-tools/blob/acbe085c54698d0ccb98892efbc233f61d2faed5/validation/util/test.go#L85

Reply:
No, here just to pass absolute cgroup path.
I will update PRs to support absolute and relative cgroup path validation